0033-relationship-between-protocols/0033-relationship-between-protocols.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Relationship between Protocols
+draft: 1
+---
+
+# Relationship between Protocols
+
+## Prerequisites
+This document assumes the reader is familiar with the following document:
+
+- [Interledger Architecture](../0001-interledger-architecture/0001-interledger-architecture.md)
+
+## Terminology
+- A **node** is a participant in an Interledger network. It may be a [connector](../0001-interledger-architecture/0001-interledger-architecture.md#connectors), a sender or a receiver.
+- A **payment** is sending value from one to another but it doesn't mean settled necessarily that is explained below. In the Interledger network, a payment could mean a **node-to-node** payment or an **end-to-end** payment.
+  - A **node-to-node** payment means the intention of transferring value of a node (a connector, an endpoint sender, or an endpoint receiver) to another node which is directly connected to the sender node.
+  - An **end-to-end** payment means the intention of transferring value of an endpoint sender to an endpoint receiver.
+- A **settlement** is ensuring that one has the paid value. For instance, executing a transaction on a certain ledger that the payee has an account on.
+  - A **node-to-node** payment is settled by means that the two nodes agreed upon. It is usually a ledger that tracks the amount that both nodes have.
+  - An **end-to-end** payment is settled through a chain of settlements of intermediaries above.
+  - Using [STREAM](../0029-stream/0029-stream.md) protocol, payments on the Interledger network could be bidirectional, thus settlements could be as well.
+  - Refer to [Interledger Architecture](../0001-interledger-architecture/0001-interledger-architecture.md) and [Perspective](#perspective).
+
+## Motivation
+Because Interledger Protocol consists of several protocols and some related specifications, it is slightly difficult to grasp the entire relationship between the protocols. It is important for those who concretely implement ILP, to explicate the relationship so that they could quickly imagine how they should go forward. This document aims to build their mental models for ILP. A mental model is what a reader assumes something to be, reading some explanation. The mental model helps the reader predicting the behavior of what is explained even if the details are not given at that time.
```

Comment:
Nitpick: I would say "Because the Interledger Protocol suite consists of several protocols and some related specifications, it can be difficult to understand the relationship between each of the protocols. This document attempts to provide an overview and mental model of how the components fit together. It is primarily intended for implementers of parts of the Interledger stack but may also be useful for others who wish to understand how Interledger works as a whole."

Reply:
I used your explanation, but left the explanation of a mental model because some readers may not know it. Is that ok?